### ISSUE TITLE
Create Template

### ISSUE THREAD

**Issue by McNerdius** [McNerdius/TailBlazorLite#1]
- [x] Add `.template.config` > `template.json` 
- [x] Restructure files/folders minimally/only as needed.
    * A template project might use subfolder(s) but i'd like this to be "clone and run"-able as well. 
 There will be files at the root level that are not part of the template (readme etc.).
- [x] Solve the extraneous files issues and segue into issue #2 by creating a `.nuspec` file, specifying ignored files.
    * Using a `nuspec` as opposed to [special csproj](https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates#packing-a-template-into-a-nuget-package-nupkg-file) bits separates concerns: the `csproj` contains only what's needed for a "clone and run" sample.

**Comment by McNerdius** [comment]
I am unable to create/Install templates using .NET 6 preview 5.  Cloning [this repository](https://github.com/sayedihashimi/template-sample) and using `dotnet new --install` as per documentation works fine using .NET 5, but using .NET 6 SDK *each file* is a "template".  Using `nuget pack` works fine but attempting to install the `nupkg` under .NET 6 results in the same "template per file" issue.

Will be looking thru release notes/issues and pondering downgrading the template to use .NET 5 in the meantime.

---

[The errors](https://pastebin.com/raw/p7YcnKY6)

**Comment by McNerdius** [comment]
found [workaround](https://github.com/dotnet/templating/issues/3378) for "install from directory" problem:

> I can reproduce this bug with dotnet new -i .\, as the workaround install your template as dotnet new -i ..\<the folder with your template>

**Comment by McNerdius** [comment]
done but merging blocked by issue #3
